rules/oe-rules/preprocessors/transpiler.js

```diff
@@ -177,7 +177,8 @@ class Transpiler {
       try {
         await fs.ensureDir(lessPath);
         files = await new Files(this.serverOptions);
-        widgetsLessFiles = await files.findFiles(["widgets"], ["less"]);
+        widgetFileGlobs = files.getWidgetFileGlobFilter();
+        widgetsLessFiles = await files.findFiles(["widgets"], ["less"], null, widgetFileGlobs);
```

Comment:
Hi @mdziadkowiec95, this might not be enough. Check my comment: https://github.com/douglashipolito/hocus-proxus-occ/pull/5#issuecomment-725760103

Reply:
@douglashipolito, committed this change with the widgets condition